Mathlib/Order/InitialSeg.lean

```diff
@@ -90,6 +102,11 @@ theorem init_iff (f : r ≼i s) {a : α} {b : β} : s b (f a) ↔ ∃ a', f a' =
     exact ⟨a', rfl, f.map_rel_iff.1 h⟩,
     fun ⟨a', e, h⟩ => e ▸ f.map_rel_iff.2 h⟩
 
+theorem init_le [Preorder α] [PartialOrder β] (f : @InitialSeg α β (· < ·) (· < ·)) {a : α} {b : β}
+    (h : b ≤ f a) : ∃ a', f a' = b := by
```

Comment:
I don't know how to understand this lemma name?

Reply:
It's a less-equal version of [`InitialSeg.init`](https://leanprover-community.github.io/mathlib4_docs/Mathlib/Order/InitialSeg.html#InitialSeg.init). But thinking about it, it might be better to restate this as `IsLowerSet (range f)`.